src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/Transport/UnixDomainSocketDataTransport.cs

```diff
@@ -43,7 +43,7 @@ internal class UnixDomainSocketDataTransport : IDataTransport, IDisposable
     {
         this.unixEndpoint = new UnixDomainSocketEndPoint(unixDomainSocketPath);
         this.timeoutMilliseconds = timeoutMilliseconds;
-        this.Connect();
+        this.CreateSocket();
```

Comment:
we can keep the same code as before, with just one change: wrap this line inside try..catch, and log the fact that UDS was unavailable at startup.